packages/database/src/migrations/20210922211939-LESMIS-number-of-schools-overlays-modifies-data.js

```diff
@@ -0,0 +1,324 @@
+'use strict';
+
+import { generateId, insertObject, codeToId, nameToId, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getReport = (reportCode, dataElements) => {
+  const aggregations = [
+    {
+      type: 'FINAL_EACH_YEAR',
+      config: {
+        dataSourceEntityType: 'sub_district',
+        aggregationEntityType: 'requested',
+      },
+    },
+  ];
+
+  if (reportCode.includes('Province')) {
+    aggregations.push({
+      type: 'SUM_PER_ORG_GROUP',
+      config: {
+        dataSourceEntityType: 'sub_district',
+        aggregationEntityType: 'district',
+      },
+    });
+  }
+
+  return {
+    code: reportCode,
+    config: {
+      fetch: {
+        dataElements,
+        aggregations,
+      },
+      transform: [
+        {
+          transform: 'insertColumns',
+          columns: {
+            '=$dataElement': '=$value',
+          },
+        },
+        {
+          transform: 'updateColumns',
+          insert: {
+            organisationUnitCode: '=$organisationUnit',
+            value: `=sum(${dataElements.map(code => `$${code}`)})`,
```

Comment:
@reviewer I'm not sure if there is an easier way to sum up all the dataElements or not?

Reply:
(Addressed in a separate comment so I could add a multiline suggestion)